AutoMl/tests/System/V1/AutoMlSmokeTest.php

```diff
@@ -68,22 +69,22 @@ public static function setUpBeforeClass()
      */
     public function testAutoMl(AutoMlClient $automl)
     {
-        $formattedParent = $automl->locationName(self::$projectId, self::$location);
-        $dataset = new Dataset([
-            'display_name' => uniqid(self::TESTING_PREFIX),
-            'translation_dataset_metadata' => new TranslationDatasetMetadata([
-                'source_language_code' => 'en',
-                'target_language_code' => 'es'
-            ])
-        ]);
+        $notFound = false;
+        $dataset = AutoMlClient::datasetName(
+            self::$projectId,
+            self::$location,
+            'TRL100'
+        );
 
-        $response = $automl->createDataset($formattedParent, $dataset);
```

Comment:
This returns an instance of `Google\LongRunning\Operation` instead of `Google\ApiCore\OperationResponse`. The GAPIC configuration appears to be off and is treating this as a normal call instead of an operations call. I'd like to get this fix out ASAP, further investigate the required GAPIC config change and ensure it is what we are looking for - then potentially follow up with a breaking change to have it return an `OperationResponse`.